fonts.css

```diff
@@ -0,0 +1,13 @@
+@font-face {
+    font-family: 'Gotham Pro';
+    src: url('fonts/GothamPro.woff') format('woff'), url('fonts/GothamPro.woff2') format('woff2');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/final-Axion/pullRequest?prid=5455661)